src/resdk/resolwe.py

```diff
@@ -200,6 +200,18 @@ def version_check(self):
                 "Warning: unable to read the minimal supported version from the server."
             )
 
+    def version_output(self):
+        """Output the version of the server modules."""
+        url = urljoin(self.url, "/about/versions")
+        try:
+            response = requests.get(url)
+            server_version = response.json()
+            return server_version
+        except Exception:
+            self.logger.warning(
```

Comment:
This hides the original exception, which may contain important information about the reason the request failed.

I suggest you replace the `self.logger.warning` with `self.logger.exception` and re-raise the exception (or create a new one and raise that one). For instance, create a ResolweServerError with the description bellow joined with the information from the original exception.